pom.xml

```diff
@@ -224,7 +224,6 @@
                 <configuration>
                     <productVersion>${bamboo.version}</productVersion>
                     <productDataVersion>${bamboo.data.version}</productDataVersion>
-                    <productDataPath>../generated-test-resources.zip</productDataPath>
```

Comment:
Removing this means you need to completely setup your test instance every time.

There are both good and bad points for this. It just gets very tiring to do every time you do a clean